com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueHistoryFrame.cs

```diff
@@ -5,9 +5,7 @@
 namespace MLAPI.Messaging
 {
     /// <summary>
-    /// QueueHistoryFrame
     /// Used by the RpcQueueContainer to hold queued RPCs
-    /// All queued Rpcs end up in a PooledNetworkBuffer within a QueueHistoryFrame instance.
     /// </summary>
     public class RpcQueueHistoryFrame
```

Comment:
I think changes in this file should go in a separate, very small PR.

Reply:
Yeah, I see your point.  I had thought about that... but the only changes are updated comments...so while it is a change... it was so minor I truly didn't think it was worth breaking into an entire separate PR for the two comment lines being deleted.

Reply:
This should stay in this update as it is required by the BufferDataValidationComponent.  I guess it does make sense in some ways to keep it separate from the unit tests, but the unit tests depend upon that change and then you have to link to the PR that requires the change so in the event anyone reverts the single and simple PR they know that they need to fix the associated unit tests otherwise their reverting of the single PR (i.e. associated with just RpcQueueHistoryFrame.cs changes) would yield failed Yamato builds, and as such they would have to merge and revert both PRs in order to remove the RpcQueueHistoryFrame.cs changes. (i.e. The BufferDataValidationComponent will fail if those changes are reverted separately).

That is why I included it in this PR, because it would be problematic to just revert that one file as an independent PR without having to jump through a few hoops first (in order to get Yamato to pass all unit tests which is required to squash and merge).

Does this make more sense as to why it was included?